common/lib/capa/capa/templates/optioninput.html

```diff
@@ -30,6 +30,6 @@
   </div>
   <p class="answer" id="answer_${id}"></p>
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="label_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
@alisan617 @cptvitamin and I just chatted about this in person.

He would like both the "message" and the "status" areas to point to the label (or legend) instead of the inputs. So the current usage on line 33 is correct, but the one on line 27 should be changed. 

And then you should go through the other templates and link both "message" and "status" to labels/legends instead of inputs.